blogger.go

```diff
@@ -0,0 +1,125 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/signal"
+	"path/filepath"
+	"strings"
+	"syscall"
+	"time"
+)
+
+var (
+	portAddr   = flag.String("http", ":8080", "HTTP listen address")
+	contentDir = flag.String("content", "content/", "Content directory")
+	index      = flag.Bool("index", false, "Show index")
+)
+
+func main() {
+	flag.Parse()
+
+	if strings.Index(*portAddr, ":") != 0 {
```

Comment:
Wouldn't it make more sense _not_ to require users to write the colon? For example, they could run the server with:

```go
./blogger -port 8888
```

Reply:
oh yeah,I will fix that.